### PR TITLE
feat: onEnter focus on editor

### PR DESCRIPTION
**Pull request by Kenny-Gin1**
- On enter in the Title input focuses, the editor now
- Autofocus the title when entering the page

### CODE REVIEW

File: apps/web/src/components/PageEditor.tsx [hunk 1]
```diff
@@ -89,10 +102,13 @@ export function PageEditor({ page, onSave, isSaving }: PageEditorProps) {
       <AuthDialog open={!isAuthenticated} />
       <div className="mb-4 flex flex-row justify-between">
         <input
+          ref={inputRef}
```
Comment (ericzakariasson) [inline apps/web/src/components/PageEditor.tsx:105]: ✨ You can actually specify `autofocus` as an attribute!

File: apps/web/src/components/PageEditor.tsx [hunk 2]
```diff
@@ -55,6 +56,10 @@ export function PageEditor({ page, onSave, isSaving }: PageEditorProps) {
     run();
   }, [ceramic]);
 
+  useEffect(() => {
+    inputRef.current?.focus();
```
Comment (ericzakariasson) [inline apps/web/src/components/PageEditor.tsx:60]: Does this work when you press `edit` as well?

**Review by ericzakariasson — APPROVED** [review]
nice one 👏  just a question

**Review by philipandersson — APPROVED** [review]
LGTM, Eric made some good comment about `autofocus`

File: apps/web/src/components/PageEditor.tsx (hunk 1 above)
Comment (Kenny-Gin1) [reply apps/web/src/components/PageEditor.tsx:105]: Yeah, I did that first, and it didn't work for some reason.  So ended up using the `inputRef` instead

**Comment by Kenny-Gin1** [comment]
I made a refactor to replace `useRef` with `useCallback` because the `autoFocus` didn't work when switching pages and on pages where the page was already created (e.g. edited pages).

**Pull request merged**